warehouse/query-core/src/test/java/datawave/query/CompositeQueryTest.java

```diff
@@ -142,4 +145,13 @@ protected void testInit() {
         this.auths = CitiesDataType.getTestAuths();
         this.documentKey = CityField.EVENT_ID.name();
     }
+    
+    @Test
+    public void testRegex() throws JavaRegexAnalyzer.JavaRegexParseException {
+        String dt = "2004-10-22T00:00:00.0Z";
+        JavaRegexAnalyzer analyzer = new JavaRegexAnalyzer(Pattern.quote(dt) + "\ffffname blah.*");
+        System.out.println(analyzer.getLeadingLiteral());
+        System.out.println(analyzer.getTrailingLiteral());
```

Comment:
Either this test should be removed, or these println statements could be changed to an assert that the leading literal is `dt + "\ffffname blah"` and that the trailing literal is null.

Reply:
Oh thanks for commenting! These weren't meant to be included. High likelihood I would have missed this even after looking at it again!